framework/resources/Datasources/table_stats/analyze_tables.sql

```diff
@@ -0,0 +1,19 @@
+alter session set `store.parquet.reader.int96_as_timestamp`=true;
+analyze table `table_stats/alltypes_with_nulls` compute statistics;
+analyze table `table_stats/date_intervals` compute statistics;
+analyze table `table_stats/impala_parquet` compute statistics;
+--analyze table `table_stats/onlynulls_parquet` compute statistics;
+analyze table `table_stats/DRILL-4735` compute statistics;
+analyze table `table_stats/inNstedDirAutoPrtn` compute statistics;
+refresh table metadata `table_stats/lineitem_parquet_partitioned_hive_date`;
```

Comment:
Any reason we have refresh table metadata in between?

Reply:
Because the test case would have been how generating stats would behave if it has a metadata cache.